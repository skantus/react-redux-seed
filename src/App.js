import React from 'react';
import Home from 'components/Home';

const App = () => <Home tech="React" />;

export default App;
