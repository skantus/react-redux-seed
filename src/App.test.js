import React from 'react';
import App from './App';

describe('<App />', () => {
  const wrapper = shallow(<App />);

  it('render <App /> component as expected', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should render without blowing up', () => {
    expect(wrapper.length).toEqual(1);
  });
});
