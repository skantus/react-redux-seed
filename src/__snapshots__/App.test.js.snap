// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> render <App /> component as expected 1`] = `
<Home
  tech="React"
/>
`;
